src/UI/SubtitleEntry.lua

```diff
@@ -120,6 +123,9 @@ function SubtitleEntry:UpdateMultipleText(): ()
                 TextTransparency = 0,
             })
             self.Window:UpdateSize()
+            if self.TypeWriterEnabled then
+                task.spawn(TypeWriter, self.TextLabel, Message, (self.Duration/Message:len()))
```

Comment:
I *think* we use spaces around `/`, so `self.Duration / Message:len())`.
I think we also use `string.len(String)` instead of `String:len()`, so I'd think `self.Duration / string.len(Message))` would be more consistent even though it is longer.

Also, does this mean the typewriter option can't handle rich text? That would need to either be handled or documented.